tools/mlir-cpu-runner/CMakeLists.txt

```diff
@@ -21,3 +21,14 @@ target_link_libraries(mlir-cpu-runner PRIVATE
   LLVMCore
   LLVMSupport
 )
+
+install(TARGETS mlir-cpu-runner
+  ${export_to_llvmexports}
+  COMPONENT mlir-cpu-runner
+  RUNTIME DESTINATION ${LLVM_TOOLS_INSTALL_DIR})
+
+if(NOT LLVM_ENABLE_IDE)
+  add_llvm_install_targets("install-mlir-cpu-runner"
+    DEPENDS mlir-cpu-runner
```

Comment:
Do we need mlir-cpu-runner?

Reply:
I focused on directories with files needed by f18.  I didn't try to cull the number of files from each of those directory to match exactly the needs of f18.  I.e. I was trying to be complete for each directory that I touched.